ServiceLambdaJavaClient/src/main/java/com/kenzie/capstone/service/client/LambdaServiceClient.java

```diff
@@ -61,7 +60,7 @@ public LambdaDrink updateDrink(LambdaDrink lambdaDrink) {
     }
 
     public String deleteDrink(String id) {
-        EndpointUtility endpointUtility = new EndpointUtility();
+        final EndpointUtility endpointUtility = new EndpointUtility();
```

Comment:
these are repeating code lines - perhaps make this a class variable?

Reply:
Made global variable